migrations/000006.php

```diff
@@ -0,0 +1,11 @@
+<?php
+// This migration adds a "activated" column to the subscriptions table and sets all values to true.
+
+/** @noinspection PhpUndefinedVariableInspection */
+$columnQuery = $db->query("SELECT * FROM pragma_table_info('subscriptions') WHERE name='activated'");
+$columnRequired = $columnQuery->fetchArray(SQLITE3_ASSOC) === false;
+
+if ($columnRequired) {
+    $db->exec('ALTER TABLE subscriptions ADD COLUMN activated BOOLEAN DEFAULT true');
+    $db->exec('UPDATE subscriptions SET activated = true');
+}

```

Comment:
Hi.
Sorry for the delay.

What I meant by Enabled is the default state of a subscription, is that the modifier should be called Disabled.
When you create a subscription it is enabled by default. By selecting something you can disable it.

So, the new column should be `disabled` and set to `false `by default.

Like, the checkbox on the form should be checked to DISABLE the subscription, not to enable it (since enabled is the default state), and so on.

Reply:
Hi @ellite ,

got it, make sense. 
Will update the code in the next days and let you know - a bit busy at the moment.

Reply:
No problem. Thank you.

Reply:
@ellite Please review